pgmpy/readwrite/XMLBIF.py

```diff
@@ -1,5 +1,12 @@
 #!/usr/bin/env python
 
+import string
+import re
+from io import BytesIO
+
+# TODO input and output state
```

Comment:
Happy to try and do that but may take a while as I don't understand how the state works in the CPD representation. Maybe better in a separate pull request? Are there any examples of creating a CPD with state (couldn't find any in existing readers)